src/gz21_ocean_momentum/utils.py

```diff
@@ -17,7 +17,7 @@
 import gz21_ocean_momentum.models as models
 import sys
 
-sys.modules["models"] = models # what are we doing here anyway
+sys.modules["models"] = models  # what are we doing here anyway
```

Comment:
Not sure what this is? Is this necessary?

Reply:
I suspect we do not need it. It was there in your initial commit in February, though.

Reply:
Hm, I was wrong. We seem to need it for the mlflow runs. It tried commenting it out, and the non-mlflow Python runs went through, but the mlflow run failed half-way through. I am running another test, and this is confirmed I'll put it back in and add a comment.